### ISSUE TITLE
geneos package install downloads unrelevant binaries

### ISSUE THREAD

**Issue by cljnie** [ITRS-Group/cordial#166]
when doing a geneos package install on a rh8 system, geneos downloads both normal and el8 versions and install both

```bash
/radmin@geneos-lab1# geneos host show localhost
[
    {
        "name": "localhost",
        "config": {
            "geneos": "/opt/itrs",
            "homedir": "/home/radmin",
            "hostname": "geneos-lab1",
            "name": "localhost",
            "os": "linux",
            "osinfo": {
                "ansi_color": "0;31",
                "bug_report_url": "https://bugzilla.redhat.com/",
                "cpe_name": "cpe:/o:centos:centos:8",
                "home_url": "https://centos.org/",
                "id": "centos",
                "id_like": "rhel fedora",
                "name": "CentOS Stream",
                "platform_id": "platform:el8",
                "pretty_name": "CentOS Stream 8",
                "redhat_support_product": "Red Hat Enterprise Linux 8",
                "redhat_support_product_version": "CentOS Stream",
                "version": "8",
                "version_id": "8"
            }
        }
    }
]
/radmin@geneos-lab1# geneos package ls
Component  Host         Version             Links        LastModified          Path
webserver  localhost    6.4.0 (latest)      active_prod  2023-07-05T11:34:23Z  /opt/itrs/packages/webserver/6.4.0
webserver  localhost    6.3.0                            2023-04-03T12:38:06Z  /opt/itrs/packages/webserver/6.3.0
netprobe   localhost    6.4.0+el8 (latest)  active_prod  2023-07-05T11:34:05Z  /opt/itrs/packages/netprobe/6.4.0+el8
netprobe   localhost    6.4.0                            2023-07-05T11:34:08Z  /opt/itrs/packages/netprobe/6.4.0
netprobe   localhost    6.3.1-el8                        2023-05-04T14:16:23Z  /opt/itrs/packages/netprobe/6.3.1-el8
licd       localhost    6.4.0+el8 (latest)  active_prod  2023-07-05T11:34:12Z  /opt/itrs/packages/licd/6.4.0+el8
licd       localhost    6.1.0-el8                        2022-12-20T15:51:53Z  /opt/itrs/packages/licd/6.1.0-el8
ac2        localhost    6.4.0 (latest)      active_prod  2023-07-05T11:33:49Z  /opt/itrs/packages/ac2/6.4.0
gateway    localhost    6.4.0+el8 (latest)  active_prod  2023-07-05T11:33:25Z  /opt/itrs/packages/gateway/6.4.0+el8
gateway    localhost    6.3.1-el8                        2023-05-04T14:08:09Z  /opt/itrs/packages/gateway/6.3.1-el8
fileagent  geneos-lab2  6.4.0 (latest)      active_prod  2023-07-05T11:34:25Z  /opt/itrs/packages/fileagent/6.4.0
licd       geneos-lab2  6.4.0+el8 (latest)  active_prod  2023-07-05T11:35:02Z  /opt/itrs/packages/licd/6.4.0+el8
licd       geneos-lab2  6.1.0                            2022-12-20T15:51:55Z  /opt/itrs/packages/licd/6.1.0
webserver  geneos-lab2  6.4.0 (latest)      active_prod  2023-07-05T11:34:44Z  /opt/itrs/packages/webserver/6.4.0
webserver  geneos-lab2  6.3.0                            2023-03-30T15:59:41Z  /opt/itrs/packages/webserver/6.3.0
netprobe   geneos-lab2  6.4.0+el8 (latest)  active_prod  2023-07-05T11:34:37Z  /opt/itrs/packages/netprobe/6.4.0+el8
netprobe   geneos-lab2  6.4.0                            2023-07-05T11:34:59Z  /opt/itrs/packages/netprobe/6.4.0
netprobe   geneos-lab2  6.3.1                            2023-05-04T14:16:41Z  /opt/itrs/packages/netprobe/6.3.1
gateway    geneos-lab2  6.4.0+el8 (latest)  active_prod  2023-07-05T11:33:28Z  /opt/itrs/packages/gateway/6.4.0+el8
gateway    geneos-lab2  6.3.1                            2023-05-04T14:08:15Z  /opt/itrs/packages/gateway/6.3.1
ac2        geneos-lab2  6.4.0 (latest)      active_prod  2023-07-05T11:34:53Z  /opt/itrs/packages/ac2/6.4.0
ac2        geneos-lab3  6.4.0 (latest)      active_prod  2023-07-05T11:35:10Z  /opt/itrs/packages/ac2/6.4.0
gateway    geneos-lab3  6.4.0+el8 (latest)  active_prod  2023-07-05T11:33:32Z  /opt/itrs/packages/gateway/6.4.0+el8
gateway    geneos-lab3  6.3.1                            2023-05-04T14:08:19Z  /opt/itrs/packages/gateway/6.3.1
webserver  geneos-lab3  6.4.0 (latest)      active_prod  2023-07-05T11:35:42Z  /opt/itrs/packages/webserver/6.4.0
webserver  geneos-lab3  6.3.0                            2023-03-30T15:59:50Z  /opt/itrs/packages/webserver/6.3.0
netprobe   geneos-lab3  6.4.0+el8 (latest)  active_prod  2023-07-05T11:35:24Z  /opt/itrs/packages/netprobe/6.4.0+el8
netprobe   geneos-lab3  6.4.0                            2023-07-05T11:35:28Z  /opt/itrs/packages/netprobe/6.4.0
netprobe   geneos-lab3  6.3.1                            2023-05-04T14:16:53Z  /opt/itrs/packages/netprobe/6.3.1
fileagent  geneos-lab3  6.4.0 (latest)      active_prod  2023-07-05T11:35:34Z  /opt/itrs/packages/fileagent/6.4.0
licd       geneos-lab3  6.4.0+el8 (latest)  active_prod  2023-07-05T11:35:31Z  /opt/itrs/packages/licd/6.4.0+el8
licd       geneos-lab3  6.1.0                            2022-12-20T15:51:57Z  /opt/itrs/packages/licd/6.1.0

```

**Comment by pgalbavy-itrs** [comment]
I cannot reproduce this in simple testing. Can you try to keep a record of the commands that result in this in the future?

```bash
[geneos@8bc05453d7ec ~]$ geneos package install netprobe
geneos-netprobe-6.4.0-el8-linux-x64.tar.gz 100% |█████████████████████████████████████████████████████████████████████████████████████████████████| (295/295 MB, 18 MB/s)         
installed "geneos-netprobe-6.4.0-el8-linux-x64.tar.gz" to "/home/geneos/packages/netprobe/6.4.0+el8"
netprobe "active_prod" on localhost updated to 6.4.0+el8
[geneos@8bc05453d7ec ~]$ geneos package ls                     
Component  Host       Version             Links        LastModified          Path
netprobe   localhost  6.4.0+el8 (latest)  active_prod  2023-07-18T13:40:31Z  /home/geneos/packages/netprobe/6.4.0+el8
```

**Comment by pgalbavy-itrs** [comment]
This was fixed around v1.18.x - closing.
